src/components/app.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import styles from './styles.css';
+
+class App extends Component {
```

Comment:
This can be done with the arrow function syntax as you had it before.
`const App = () => (
    <div className="app-blue">
        <h1>Hello, emilio</h1>
    </div>
)`